examples/react/file-upload/src/ContentPage.jsx

```diff
@@ -2,10 +2,11 @@ import React, { useState } from 'react'
 import { useUploader } from '@w3ui/react-uploader'
 import { withIdentity } from './components/Authenticator'
 import './spinner.css'
+import Loader from './components/Loader'
 
 export function ContentPage () {
-  const [{ storedDAGShards }, uploader] = useUploader()
-  const [file, setFile] = useState(null)
+  const [{ storedDAGShards, progressStatuses }, uploader] = useUploader()
```

Comment:
Nit: I find plural naming confusing personally, I would probably called it uploadProgress instead personally

Reply:
ahh, I agree and like that name - should be pretty quick to change since it's only in this codebase, thanks!